apm-agent-plugins/apm-cassandra/apm-cassandra4-plugin/src/test/java/co/elastic/apm/agent/cassandra/Cassandra4InstrumentationIT.java

```diff
@@ -47,12 +43,9 @@
 @Testcontainers
 class Cassandra4InstrumentationIT extends AbstractInstrumentationTest {
 
-    private static final Logger logger = LoggerFactory.getLogger(Cassandra4InstrumentationIT.class);
-
     @Container
     public static GenericContainer<?> cassandra = new GenericContainer<>("cassandra:4.0")
         .withExposedPorts(9042)
-        .withLogConsumer(new Slf4jLogConsumer(logger))
```

Comment:
Why can't we use slf4j in integration tests?
Is implementing a custom log consumer an option?

Reply:
We probably can use slf4j, I didn't think it worth the possible confusion (meaning - have to be aware of which `LoggerFactory` you use).
Since it is now packaged into the core anyway, let me add a utility that knowingly creates this consumer properly and restore this functionality.

Reply:
Restored